### ISSUE TITLE
Training loss keeps going up

### ISSUE THREAD

**Issue by Valeyards** [cwmok/LapIRN#13]
Hi @cwmok , I'm trying to train LapIRN in l2r 2020 task2 i.e., the inhale-exhale lung registration.
I'm uisng Train_LapIRN_disp.py. During training procedure, the initial loss looks like
![image](https://user-images.githubusercontent.com/36869557/169764369-ec1b0409-56bc-4b90-8c13-1c537dc4d079.png)

After some iterations it becomes...
![image](https://user-images.githubusercontent.com/36869557/169765533-426f9da2-91ef-42c9-aa25-8a89382ab255.png)

The lr is default 1e-4 since I found if I change it to 1e-3 the situation is even worse.
![image](https://user-images.githubusercontent.com/36869557/169764616-d32ca483-5bcd-40b2-8095-0ed25e46087a.png)

I only did some minor changes on Dataset_Epoch in Functions.py and adjust the input shape to (192, 192, 208) as it is the original size right?
![image](https://user-images.githubusercontent.com/36869557/169764786-2525081c-b2c6-477a-92f9-6d825bd08b6c.png)

I'm using torch 1.7.1+cuda 11.0 on a single RTX 3090
Do you have any idea about this?

**Comment by cwmok** [comment]
1. We trained our model at half-resolution, i.e., 96x96x104.
2. We applied CT windowing - [100, 1518] to both scans.
3. You should define the region of interest (ROI) of the registration using masks (Cost function masking). For masking, please refer to the "Masks" section in https://simpleelastix.readthedocs.io/Introduction.html.
4. We found that NGF is more stable than NCC in this task. Yet, NCC yields slightly better performance when it works.

Other minor changes are summarized in the ppt slides below:
[Learn2Reg2021_Tony_oral.pdf](https://github.com/cwmok/LapIRN/files/8752483/Learn2Reg2021_Tony_oral.pdf)

**Comment by Valeyards** [comment]
> 1. We trained our model at half-resolution, i.e., 96x96x104.
> 2. We applied CT windowing - [100, 1518] to both scans.
> 3. You should define the region of interest (ROI) of the registration using masks (Cost function masking). For masking, please refer to the "Masks" section in https://simpleelastix.readthedocs.io/Introduction.html.
> 4. We found that NGF is more stable than NCC in this task. Yet, NCC yields slightly better performance when it works.
> 
> Other minor changes are summarized in the ppt slides below: [Learn2Reg2021_Tony_oral.pdf](https://github.com/cwmok/LapIRN/files/8752483/Learn2Reg2021_Tony_oral.pdf)

Thank you!
